### PR TITLE
Simple Pet Shop correction of handful of links to use anchors inside p tags

### PR DESCRIPTION
**Pull request by mikedotexe**
In the Pet Shop tutorial there are a few cases where there is a pattern like this inside the markdown:

```html
<p class="alert alert-info">
  Content [markdown link](/relative/path)
</p>
```

These links won't show up so I turned anchors. Then made sure the links worked, and correct the ones that were off to their proper location.

### CODE REVIEW

**Comment by eggplantzzz** [comment]
Oh wow, thanks for catching this @mikedotexe! It seems like a lot of these links are broken, I'm going to ask some colleagues if they know what is going on because I'm not sure the proper fix is to convert these to anchor tags. I think these used to work, I wonder what happened to break them...

**Comment by mikedotexe** [comment]
> Oh wow, thanks for catching this @mikedotexe! It seems like a lot of these links are broken, I'm going to ask some colleagues if they know what is going on because I'm not sure the proper fix is to convert these to anchor tags. I think these used to work, I wonder what happened to break them...

Sure thing, yes we've had the same problem over here (we use Docusaurus) and this happens when you mix html and markdown. Glad to help out even in little ways

File: src/tutorials/pet-shop.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ Now we are ready to create our own migration script.
 1. Before we can migrate our contract to the blockchain, we need to have a blockchain running. For this tutorial, we're going to use [Ganache](/ganache), a personal blockchain for Ethereum development you can use to deploy contracts, develop applications, and run tests. If you haven't already, [download Ganache](/ganache) and double click the icon to launch the application. This will generate a blockchain running locally on port 7545.
 
    <p class="alert alert-info">
-     <strong>Note</strong>: Read more about Ganache in the [Truffle documentation](/docs/ganache/using).
+     <strong>Note</strong>: Read more about Ganache in the <a href="/docs/ganache/overview">Truffle documentation</a>.
```
Comment (cds-amal) [inline src/tutorials/pet-shop.md:226]:
```suggestion
     <strong>Note</strong>: Read more about Ganache in the <a href="/docs/ganache/using">Truffle documentation</a>.
```
The `/docs/ganache/using` resolves to a Ganache Quick Start guide. The overview might be too general for someone going through PetShop.

**Comment by mikedotexe** [comment]
Thanks for jumping on and correcting that, just getting home now. Looks all good. Look forward to have NEAR and Truffle work together on projects in the future

**Comment by cds-amal** [comment]
> Thanks for jumping on and correcting that, just getting home now. Looks all good. Look forward to have NEAR and Truffle work together on projects in the future

No worries! According to the contributing guide you can press the merge button :)

**Comment by mikedotexe** [comment]
Thanks @cds-amal , looks like this is the part of the contributing guide you're talking about:
https://github.com/trufflesuite/trufflesuite.com/blob/develop/CONTRIBUTING.md#merging-processes
However, this is what I'm seeing, with no merge button.

Screen Shot 2020-08-31 at 3 35 36 PM

**Comment by eggplantzzz** [comment]
Gotcha covered!

**Pull request merged**